mud-contracts/world/src/modules/smart-deployable/systems/SmartDeployable.sol

```diff
@@ -365,7 +378,9 @@ contract SmartDeployable is EveSystem, SmartDeployableErrors {
       entityId
     );
     // elapsed time in seconds, multiplied by consumption rate per second
-    uint256 fuelConsumed = (block.timestamp - data.lastUpdatedAt) * (data.fuelConsumptionPerMinute / 60);
+    uint256 fuelConsumed = (((block.timestamp - data.lastUpdatedAt) * (10 ** DECIMALS)) /
+      (data.fuelConsumptionPerMinute));
```

Comment:
Is it possible to store `fuelConsumptionPerMinute` in a new `oneFuelUnitConsumptionIntervalInSec` variable at the beginning of ` _currentFuelAmount`? 
And we can add a comment on the assignment to explain why

Reply:
fixed in [e72f25d](https://github.com/projectawakening/world-chain-contracts/pull/114/commits/e72f25dffac755c0c835ad7ac3f380d680994438)